train.py

```diff
@@ -242,6 +263,33 @@ def main(
                 label_types=parse_label_types(label_types),
                 clear_dataset_cache=clear_dataset_cache,
                 title=title,
+                disable_wandb=disable_wandb,
+            )
+        if (
+            content_input == ContentInput.Radio
+            and input_type == InputType.Image
+            and ml_task == MLTask.Classification
+        ):
+
+            assets = get_assets(
+                kili,
+                project_id,
+                parse_label_types(label_types),
+                labeling_statuses=["LABELED"],
+            )
+            assets = assets[:max_assets] if max_assets is not None else assets
+
+            training_loss = train_and_get_error_image_classification(
```

Comment:
Would it be difficult to put  the code in a `PytorchVisionImageClassificationModel`  class to write:
```
training_loss = PytorchVisionImageClassificationModel(
                project_id, api_key, api_endpoint
            ).train(
                assets=assets,
                job=job,
                job_name=job_name,
                model_framework=model_framework,
                model_name=model_name,
                clear_dataset_cache=clear_dataset_cache,
                disable_wandb=disable_wandb,
            )
```
? If this is, we will keep it as is an create a high priority technical debt ticket.